helm/chart-operator/templates/configmap.yaml

```diff
@@ -16,6 +16,12 @@ data:
         address: 'http://0.0.0.0:{{ .Values.pod.port }}'
     service:
       helm:
+        http:
+          {{ if (.Values.Installation) }}
+          clientTimeout: '{{ .Values.Installation.V1.Helm.HTTP.ClientTimout }}'
+          {{ else }}
+          clientTimeout: '{{ .Values.helm.http.clientTimeout }}'
```

Comment:
Just thinking if we can't make use of the `ci` folder. Uncertain how you actually want to default it and in which circumstances but the other PRs I saw around this suggested it is about testing. Your call.

Reply:
It isn't for testing but the values will need to work with the integration tests.

The issue is once the operator is flattened it will be deployed from the control-plane catalog in CPs and the default catalog in tenants. They use different values structures.

I'm testing in giraffe but this isn't working. I'll likely simplify this and sort it out as part of the flattening process.